src/components/pages/InstructorNewsFeed/NewsFeedPutModal.js

```diff
@@ -0,0 +1,117 @@
+import React, { useState,useEffect } from 'react';
+import { Form, Input, Button } from 'antd';
+import axios from 'axios';
+import '../../../styles/InstructorStyles/index.less';
+import { CloseOutlined } from '@ant-design/icons';
+export default function NewsfeedPutModal(props) {
+  const { setPostOptions, postID } = props;
+  const token = JSON.parse(localStorage.getItem('okta-token-storage'));
+  const config = {
+    headers: { Authorization: `Bearer ${token.idToken.value}` },
+  };
+  const [formValues, setFormValues] = useState({
+    link: '',
+    description: '',
+    title: '',
+  });
+  
+  useEffect(()=>{
+    axios
+      .get(`https://coder-heroes-api.herokuapp.com/news/${postID}`, config)
+      .then(resp => {
+        setFormValues({
+          ...formValues,
+          link:resp.data.link,
+          description:resp.data.description,
+          title: resp.data.title
+        });
+      })
+      .catch(err => {
+        console.log('error fetching newsfeed');
```

Comment:
Did you want to change this error message?

Reply:
changed to console.error(err)